_posts/2023-02-10-dicectf2023-parallelism_and_not-baby-parallelism.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "DiceCTF 2023: parallelism and not-baby-parallelism writeup"
+external_url: "https://ik0ri4n.de/dice-ctf"
```

Comment:
Could you update this to dice-ctf-23? I added a redirect, just in case, but I guess it would be better to use that one wherever possible to avoid confusion with future DiceCTFs

Reply:
I tried to do so myself but somehow can't pull this branch